README.md

```diff
@@ -26,7 +26,7 @@ The Infura Project ID will enable the network status progress bar showing the ba
 REACT_APP_INFURA_PROJECT_ID=your-infura-project-id-here
 ```
 
-Note: The Portis wallet option is not available when running locally
+Note: The Portis wallet option is currently not available
```

Comment:
I don't think we even really need to have this here, I think we can just nuke Portis & note something about removing it in a changelog & commit history